log/default_test.go

```diff
@@ -12,7 +12,7 @@ import (
 
 const work = "work"
 
-func initDefaultLogger() {
+func init() {
```

Comment:
maybe should use TestMain instead of init?

Reply:
How? 🤔

Reply:
> How? 🤔

Nevermind, current behavior seems OK